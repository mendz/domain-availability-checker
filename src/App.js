import React from 'react';
import DomainList from './containers/DomainList/DomainList';

import './App.css';

function App() {
  return (
    <div className="App">
      <p>Please enter a list of domains you wish to check.</p>
      <p>The domains need to be separate by comma or new line</p>
      <DomainList />
    </div>
  );
}

export default App;
