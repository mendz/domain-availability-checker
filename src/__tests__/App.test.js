import React from 'react';
import toJSON from 'enzyme-to-json';
import { shallow } from 'enzyme';

import App from '../App';

describe('<App/>', () => {
  it('renders', () => {
    shallow(<App />);
  });

  it('matches the snapshot', () => {
    const wrapper = shallow(<App />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
