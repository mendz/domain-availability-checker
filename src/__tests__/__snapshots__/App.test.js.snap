// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> matches the snapshot 1`] = `
<div
  className="App"
>
  <BrowserRouter>
    <Switch>
      <Route
        component={[Function]}
        path="/history"
      />
      <Route
        component={
          Object {
            "$$typeof": Symbol(react.memo),
            "WrappedComponent": [Function],
            "compare": null,
            "displayName": "Connect(_temp)",
            "type": [Function],
          }
        }
        exact={true}
        path="/"
      />
      <Redirect
        to="/"
      />
    </Switch>
  </BrowserRouter>
</div>
`;
